src/app/static/src/tests/components/header/userHeader.test.ts

```diff
@@ -80,6 +91,7 @@ describe("user header", () => {
     });
 
     it("contains help document links", () => {
+        const store = createStore()
         // Reset translations
         registerTranslations(store);
```

Comment:
now that you've refactored the test setup, this line registering translations is obsolete and can be deleted